README.md

```diff
@@ -19,6 +19,13 @@ After downloading the zip from above:
 2. Follow the instructions in "install-mac.txt" or "install-win.txt", depending
    on what operating system you're using.
 
+Inter UI has also been packaged by many third parties:
+
+- [inter-ui on npm](https://www.npmjs.com/package/inter-ui)
+- [font-inter in Homebrew Cask Fonts](https://github.com/Homebrew/homebrew-cask-fonts)
```

Comment:
It may be helpful to monospace the keywords in these two bullet points. I.e. ``inter-ui` on npm` and ``font-inter` in Homebrew's Fonts cask`

Nit: "Homebrew Cask Fonts" → "Homebrew's Fonts cask"